include/fast_float/parse_number.h

```diff
@@ -0,0 +1,230 @@
+#ifndef FASTFLOAT_PARSE_NUMBER_H
+#define FASTFLOAT_PARSE_NUMBER_H
+
+#include "ascii_number.h"
+#include "decimal_to_binary.h"
+#include "digit_comparison.h"
```

Comment:
These need to be standard full boost paths, e.g. <boost/json/ umm... wait a minute you have this in <boost/fastfloat/> ? That's not allowed

Reply:
Turns out fast_float isn't permissive enough, so I had to remove it anyway.